doc/SAI-Proposal-PoE.md

```diff
@@ -0,0 +1,591 @@
+PoE support for SAI
+
+| Title       | PoE                    |
+| ----------- | ---------------------- |
+| Authors     | Daniela Murin, Marvell |
+| Status      | In review              |
+| Type        | Standards track        |
+| Created     | 02/25/2024             |
+| SAI-Version | 1.13                   |
+
+# Overview
+
+Power over Ethernet, or PoE, describes any of the 802.3(AF, AT, BT) standards that deliver electrical power along with data on twisted-pair Ethernet cables. This allows a single cable to provide both data connection and electricity to power the networked devices such as wireless access points (WAPs), IP cameras and VoIP phones etc.
+
+# Proposal
+
+The current SAI does not support PoE. We will implement PoE in a way that any vendor can provide their own PoE hardware and software solution.
+
+## Abbreviations and Information
+
+| **Term** | **Definition**                                                                              |
+| -------------- | ------------------------------------------------------------------------------------------------- |
+| PoE            | Power over Ethernet                                                                                  |
+| PSE            | Power Sourcing Equipment                                                                          |
+| PD             | Powered Device, can power the device through an PD-capable Ethernet port                          |
+| PoE Port       | Ethernet port that can deliver power and info to devices, located on the front panel of the board |
+
+# Key Ideas behind PoE
+
+1. PoE provides a hardware solution for providing a data connection and electricity to power connected devices
+2. The PoE software provides an interface to configure (program) and query the PoE controller and its internal peripherals such as ports etc
+3. Users can see PoE information and configure PoE features according to their needs
+
+# PoE Hardware Design Example
+
+![poe-physical-design](figures/PoE_Physical_Design.png)
+
+# SAI POE Components
+
+## New PoE SAI objects
+
+Three new SAI objects are introduced.
+
+### PoE Device ID
+
+This object is used to hold the global configuration and PoE hardware status. In many cases, one PoE device instance will have access to all the PoE hardware and components.
```

Comment:
The PoE device represents the PoE subsystem(s). 
It can be a singleton, or you can make as many SAI_OBJECT_TYPE_POE_DEVICE objects as required by your system.